### PR TITLE
update getUserMedia errors in chrome

### PR DESCRIPTION
**Pull request by tgabi333**
See: https://groups.google.com/d/msg/discuss-webrtc/DMPTKcXVUQ4/85FaASoFBwAJ

**Description**

**Purpose**

### CODE REVIEW

File: src/js/chrome/getusermedia.js [hunk 1]
```diff
@@ -136,12 +136,17 @@ module.exports = function(window) {
     return {
       name: {
         PermissionDeniedError: 'NotAllowedError',
-        InvalidStateError: 'NotReadableError',
+        PermissionDismissedError: 'NotAllowedError',
+        InvalidStateError: 'NotAllowedError',
         DevicesNotFoundError: 'NotFoundError',
         ConstraintNotSatisfiedError: 'OverconstrainedError',
+        ConstraintsNotSatisfiedError: 'OverconstrainedError',
```
Comment (fippo) [inline src/js/chrome/getusermedia.js]: does this one exist actually? https://bugs.chromium.org/p/chromium/issues/detail?id=721412#c1 looks like I searched the source and it did not.

**Comment by fippo** [comment]
@guidou can you please take a look?

File: src/js/chrome/getusermedia.js (hunk 1 above)
Comment (guidou) [reply src/js/chrome/getusermedia.js]: ConstraintsNotSatisfiedError doesn't exist and has never existed. The one that exists in older versions is ConstraintNotSatisfiedError.

Comment (tgabi333) [reply src/js/chrome/getusermedia.js]: Removed, thanks, it was copied from PSA

**Comment by fippo** [comment]
@KaptenJansson can you kick tests a couple of times?

**Review by KaptenJansson — APPROVED** [review]

**Pull request merged**